playwright/page-objects/cluster-list-page.ts

```
@@ -195,10 +195,25 @@ export class ClusterListPage extends BasePage {
-    const clusterLink = this.page.getByRole('link', { name: clusterName, exact: true });
-    await expect(clusterLink).toBeVisible({ timeout: 30000 });
-    await clusterLink.click();
+  async openClusterDefinition(
+    clusterName: string,
+    matchType: 'exact' | 'partial' | 'startsWith' = 'startsWith',
+  ): Promise<void> {
+    let clusterLink;
+    switch (matchType) {
+      case 'startsWith':
+        // Escape so clusterName is matched literally (e.g. '.' is not "any character")
+        const escapedName = clusterName.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)